models/public/conv-deconvnet/conv-deconvnet.md

```diff
@@ -0,0 +1,143 @@
+# facies-classification 
+
+## Use Case and High-Level Description
+
+The `conv-decondnet` model is designed to perform facies classification. The model has been pretrained on [this](https://drive.google.com/drive/folders/0B7brcf-eGK8CbGhBdmZoUnhiTWs) dataset. For details about the dataset and model, check out the [Original model repository](https://github.com/yalaudah/facies_classification_benchmark).
```

Comment:
What is the name of dataset?

Reply:
Also it is important to provide link on paper (https://arxiv.org/abs/1901.07659)